wagtail/wagtailimages/tests/tests.py

```diff
@@ -81,6 +81,12 @@ def test_image_tag_with_filters(self):
         self.assertTrue('width="400"' in result)
         self.assertTrue('height="300"' in result)
 
+    def test_image_tag_with_chained_filters(self):
+        result = self.render_image_tag(self.image, 'width-400|height-150')
+        self.assertTrue('width="200"' in result)
+        self.assertTrue('height="150"' in result)
```

Comment:
Actually, the test is wrong anyway :) It should be `fill-200x200 height-150` (with a space). In the image-tag, the `|` shouldn't be used, as it's something for djangos filters.